packages/file_selector/file_selector_ios/ios/Classes/messages.g.h

```diff
@@ -17,21 +19,18 @@ NS_ASSUME_NONNULL_BEGIN
 /// `init` unavailable to enforce nonnull fields, see the `make` class method.
 - (instancetype)init NS_UNAVAILABLE;
 + (instancetype)makeWithUtis:(NSArray<NSString *> *)utis
-         allowMultiSelection:(NSNumber *)allowMultiSelection;
-@property(nonatomic, strong) NSArray<NSString *> *utis;
-@property(nonatomic, strong) NSNumber *allowMultiSelection;
+    allowMultiSelection:(NSNumber *)allowMultiSelection;
+@property(nonatomic, strong) NSArray<NSString *> * utis;
```

Comment:
You need to format the PR: `dart run script/tool/bin/flutter_plugin_tools.dart format`

Reply:
done